admin_guide/cluster_capacity.adoc

```diff
@@ -0,0 +1,256 @@
+[[admin-guide-cluster-capacity]]
+= Analyzing Cluster Capacity
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As a cluster administrator, you can use the cluster capacity tool to monitor a
+node's available resources in order to increase the current resources before
```

Comment:
Its not right to say `a node's available resource` as that can be found by `oc describe node <node>` . With cluster-capacity, it gives global view of how many more pods can be scheduled based on input pod spec.